src/core/templates/show-people.js

```diff
@@ -62,113 +81,217 @@ const orcidIcon = () => html`<svg
+}
+
+function personToHTML(person) {
   const l10n = localizationStrings[lang];
-  return persons.map(getItem);
-
-  function getItem(p) {
-    const personName = [p.name]; // treated as opt-in HTML by hyperHTML
-    const company = [p.company];
-    const editorid = p.w3cid ? parseInt(p.w3cid, 10) : null;
-    /** @type {HTMLElement} */
-    const dd = html`<dd
-      class="p-author h-card vcard"
-      data-editor-id="${editorid}"
-    ></dd>`;
-    const span = document.createDocumentFragment();
-    const contents = [];
-    if (p.mailto) {
-      contents.push(html`<a
-        class="ed_mailto u-email email p-name"
-        href="${`mailto:${p.mailto}`}"
+  const personName = [person.name]; // treated as opt-in HTML by hyperHTML
+  const company = [person.company];
+  const editorId = person.w3cid ? parseInt(person.w3cid, 10) : null;
+  const contents = [];
+  if (person.mailto) {
+    contents.push(html`<a
+      class="ed_mailto u-email email p-name"
+      href="${`mailto:${person.mailto}`}"
+      >${personName}</a
+    >`);
+  } else if (person.url) {
+    contents.push(
+      html`<a class="u-url url p-name fn" href="${person.url}"
         >${personName}</a
-      >`);
-    } else if (p.url) {
-      contents.push(
-        html`<a class="u-url url p-name fn" href="${p.url}">${personName}</a>`
-      );
-    } else {
-      contents.push(html`<span class="p-name fn">${personName}</span>`);
-    }
-    if (p.orcid) {
-      contents.push(
-        html`<a class="p-name orcid" href="${p.orcid}">${orcidIcon()}</a>`
-      );
-    }
-    if (p.company) {
-      if (p.companyURL) {
-        contents.push(
-          html`
-            (<a class="p-org org h-org h-card" href="${p.companyURL}"
-              >${company}</a
-            >)
-          `
+      >`
+    );
+  } else {
+    contents.push(html`<span class="p-name fn">${personName}</span>`);
+  }
+  if (person.orcid) {
+    contents.push(
+      html`<a class="p-name orcid" href="${person.orcid}">${orcidIcon()}</a>`
+    );
+  }
+  if (person.company) {
+    const hCard = "p-org org h-org";
+    const companyElem = person.companyURL
+      ? html`<a class="${hCard}" href="${person.companyURL}">${company}</a>`
+      : html`<span class="${hCard}">${company}</span>`;
+    contents.push(html` (${companyElem})`);
+  }
+  if (person.note) contents.push(document.createTextNode(` (${person.note})`));
+  if (person.extras) {
+    person.extras
+      .map(extra => html`, ${renderExtra(extra)}`)
+      .reduce((contents, html) => {
+        contents.push(html);
+        return contents;
+      }, contents);
+  }
+  if (person.retiredDate) {
+    const { retiredDate } = person;
+    const time = html`<time datetime="${retiredDate}"
+      >${humanDate(retiredDate)}</time
+    >`;
+    contents.push(html` - ${l10n.until(time)} `);
+  }
+  const dd = html`<dd
+    class="editor p-author h-card vcard"
+    data-editor-id="${editorId}"
+  >
+    ${contents}
+  </dd>`;
+  return dd;
```

Comment:
```suggestion
  return html`<dd
    class="editor p-author h-card vcard"
    data-editor-id="${editorId}"
  >
    ${contents}
  </dd>`;
```

Reply:
I like to keep this one as seperate variable. Having gone through this process already, it's annoying to debug otherwise.